spec/support/features/in_person_helper.rb

```diff
@@ -141,6 +142,7 @@ def expect_in_person_step_indicator_current_step(text)
     expect(page).to have_css(
       '.step-indicator__step',
       text: t('step_indicator.flows.idv.find_a_post_office'),
+      wait: 10,
     )
```

Comment:
I'm wondering if we could move the `wait` somewhere a little more precise to the actual user interaction that requires an asynchronous delay. I notice we're also changing logic around `click_spinner_button_and_wait`. Ideally it'd be good if we could use that helper and avoid adding the `wait` to these other assertions, particularly on screens where we're not expecting a wait.